sql/core/benchmarks/CollationBenchmark-jdk21-results.txt

```diff
@@ -1,54 +1,88 @@
+UTF8_BINARY                                           2731           2733           2          0.0       27313.6       1.0X
+UTF8_LCASE                                            4611           4619          11          0.0       46111.4       1.7X
+UNICODE                                              28149          28211          88          0.0      281486.8      10.3X
+UNICODE_CI                                           27535          27597          89          0.0      275348.4      10.1X
 
-OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.5.0-1025-azure
-AMD EPYC 7763 64-Core Processor
+OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.8.0-1017-aws
+Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz
 collation unit benchmarks - hashFunction:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns) Relative time
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY                                        2808           2808           1          0.0       28076.2       1.0X
-UTF8_LCASE                                         5409           5410           0          0.0       54093.0       1.9X
-UNICODE                                           67930          67957          38          0.0      679296.7      24.2X
-UNICODE_CI                                        56004          56005           1          0.0      560044.2      19.9X
+UTF8_BINARY                                        4603           4618          22          0.0       46031.3       1.0X
+UTF8_LCASE                                         9510           9518          11          0.0       95097.7       2.1X
+UNICODE                                          135718         135786          97          0.0     1357176.2      29.5X
+UNICODE_CI                                       113715         113819         148          0.0     1137145.8      24.7X
 
-OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.5.0-1025-azure
-AMD EPYC 7763 64-Core Processor
+OpenJDK 64-Bit Server VM 21.0.5+11-LTS on Linux 6.8.0-1017-aws
+Intel(R) Xeon(R) CPU E5-2686 v4 @ 2.30GHz
 collation unit benchmarks - contains:     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns) Relative time
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY                                        1612           1614           2          0.1       16118.8       1.0X
-UTF8_LCASE                                        14509          14526          23          0.0      145092.7       9.0X
-UNICODE                                          308136         308631         700          0.0     3081364.6     191.2X
-UNICODE_CI                                       314612         314846         330          0.0     3146120.0     195.2X
+UTF8_BINARY                                       14957          14959           4          0.0      149568.0       1.0X
```

Comment:
We recently merged a fix for these benchmarks here https://github.com/apache/spark/pull/48804, so this regression here is outdated.

Could you please sync with mater and re-run the benchmarks to not commit the outdate results?

Reply:
Fixed